### PR TITLE
Move to an 8.0.100-alpha SDK

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

**Comment by dougbu** [comment]
Another trial. If this doesn't work, either I'll fix the problem(s) or #98072 will flow naturally into main.

**Review by wtgodbe — APPROVED** [review]

File: global.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-rtm.22471.17"
+    "version": "8.0.100-alpha.1.22477.14"
```
Comment (dougbu) [inline global.json:3]:
```suggestion
    "version": "8.0.100-alpha.1.22478.15"
```

File: global.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   },
   "tools": {
-    "dotnet": "7.0.100-rtm.22471.17",
+    "dotnet": "8.0.100-alpha.1.22477.14",
```
Comment (dougbu) [inline global.json:6]:
```suggestion
    "dotnet": "8.0.100-alpha.1.22478.15",
```

**Comment by JamesNK** [comment]
You've run into the same issue as here: https://github.com/dotnet/aspnetcore/pull/44066

**Comment by dougbu** [comment]
> You've run into the same issue as here: #44066
> `EnablePreviewFeatures` doesn't appear to work anymore.

Yup ☹️ I'll look into workarounds.

**Pull request merged**